winpr/libwinpr/sspi/sspi_winpr.c

```diff
@@ -182,18 +182,22 @@ int sspi_ContextBufferAllocTableGrow()
 	return 1;
 }
 
-void sspi_ContextBufferAllocTableFree()
+static void sspi_ContextBufferAllocTableFree(void)
 {
-	ContextBufferAllocTable.cEntries = ContextBufferAllocTable.cMaxEntries = 0;
-	free(ContextBufferAllocTable.entries);
+	if (ContextBufferAllocTable.cEntries == 0)
```

Comment:
This looks wrong ... why would we want to free the table only if there are no entries in it?